config/scenario_config_coupled_NGFS_v3.csv

```diff
@@ -1,54 +1,54 @@
-h_cpol_d95high_bIT;d95high;h_cpol_bIT;;short;SSP2|NPI;y2150;2;;h_ndc_d95high_bIT;SSP2-Base_bIT;
-o_1p5c_d50high_bIT;d50high;o_1p5c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50high_bIT;SSP2-Base_bIT;
-o_1p5c_d95high_bIT;d95high;o_1p5c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95high_bIT;SSP2-Base_bIT;
-o_2c_d50high_bIT;d50high;o_2c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50high_bIT;SSP2-Base_bIT;
-o_2c_d95high_bIT;d95high;o_2c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95high_bIT;SSP2-Base_bIT;
-d_delfrag_d50high_bIT;d50high;d_delfrag_bIT;;short;SSP2|NDC;y2030;2;;h_cpol_d50high_bIT;SSP2-Base_bIT;
-d_delfrag_d95high_bIT;d95high;d_delfrag_bIT;;short;SSP2|NDC;y2030;2;;h_cpol_d95high_bIT;SSP2-Base_bIT;
-d_rap_d50high_bIT;d50high;d_rap_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50high_bIT;SSP2-Base_bIT;
-d_rap_d95high_bIT;d95high;d_rap_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95high_bIT;SSP2-Base_bIT;
-o_cba_d50high_bIT;0;;;;;;;;;;
-o_cba_d95high_bIT;0;;;;;;;;;;
-SSP2-Base_d50_bIT;d50;SSP2-Base_bIT;;priority;SSP2|NPI;y2150;;;SSP2-Base_bIT;SSP2-Base_bIT;
-SSP2-Base_d95_bIT;d95;SSP2-Base_bIT;;priority;SSP2|NPI;y2150;;;SSP2-Base_bIT;SSP2-Base_bIT;
-h_ndc_d50_bIT;d50;h_ndc_bIT;;priority;SSP2|NDC;y2150;2;;SSP2-Base_bIT;SSP2-Base_bIT;
-h_ndc_d95_bIT;d95;h_ndc_bIT;;priority;SSP2|NDC;y2150;2;;SSP2-Base_bIT;SSP2-Base_bIT;
-h_cpol_d50_bIT;d50;h_cpol_bIT;;short;SSP2|NPI;y2150;2;;h_ndc_d50_bIT;SSP2-Base_bIT;
-h_cpol_d95_bIT;d95;h_cpol_bIT;;short;SSP2|NPI;y2150;2;;h_ndc_d95_bIT;SSP2-Base_bIT;
-o_1p5c_d50_bIT;d50;o_1p5c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50_bIT;SSP2-Base_bIT;
-o_1p5c_d95_bIT;d95;o_1p5c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95_bIT;SSP2-Base_bIT;
-o_2c_d50_bIT;d50;o_2c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50_bIT;SSP2-Base_bIT;
-o_2c_d95_bIT;d95;o_2c_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95_bIT;SSP2-Base_bIT;
-d_delfrag_d50_bIT;d50;d_delfrag_bIT;;short;SSP2|NDC;y2030;2;;h_cpol_d50_bIT;SSP2-Base_bIT;
-d_delfrag_d95_bIT;d95;d_delfrag_bIT;;short;SSP2|NDC;y2030;2;;h_cpol_d95_bIT;SSP2-Base_bIT;
-d_rap_d50_bIT;d50;d_rap_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d50_bIT;SSP2-Base_bIT;
-d_rap_d95_bIT;d95;d_rap_bIT;;short;SSP2|NDC;y2020;2;;h_ndc_d95_bIT;SSP2-Base_bIT;
-o_cba_d50_bIT;0;;;;;;;;;;
-o_cba_d95_bIT;0;;;;;;;;;;
+title;start;oldrun;path_report;qos;magpie_scen;no_ghgprices_land_until;cm_nash_autoconverge_lastrun;path_gdx;path_gdx_ref;path_gdx_bau
+SSP2-Base;0;;;medium;SSP2|NPI;y2150;;;;
+# h_ndc;0;;;medium;SSP2|NDC;y2150;2;;;
+# h_cpol;0;;;medium;SSP2|NPI;y2150;2;;;
+# o_1p5c;0;;;medium;SSP2|NDC;y2020;2;;;
+# o_2c;0;;;medium;SSP2|NDC;y2020;2;;;
+# d_delfrag;0;;;medium;SSP2|NDC;y2030;2;;;
+# d_rap;0;;;medium;SSP2|NDC;y2020;2;;;
+# SSP2-Base_bIt;0;;;medium;SSP2|NPI;y2150;;;;
+# h_ndc_bIt;0;;;medium;SSP2|NDC;y2150;2;;;
+# h_cpol_bIt;0;;;medium;SSP2|NPI;y2150;2;;;
+# o_1p5c_bIt;0;;;medium;SSP2|NDC;y2020;2;;;
+# o_2c_bIt;0;;;medium;SSP2|NDC;y2020;2;;;
+# d_delfrag_bIt;0;;;medium;SSP2|NDC;y2030;2;;;
+# d_rap_bIt;0;;;medium;SSP2|NDC;y2020;2;;;
+SSP2-Base_bIT;bIT;SSP2-Base_bIT;;priority;SSP2|NPI;y2150;;;;
```

Comment:
what does `start = bIT` even do?

Reply:
If you run `./start.R config/scenario_config_NGFS_v3.csv`, the scripts asks you which runs you want to start, and allows to start the `bIT` group or some other of these groups. This happens if either no scenario has `start = 1`, or if you pass the `--interactive` flag (say: `./start.R --interactive config/scenario_config_rev3p0p0.csv`. But you are right, it should be explained in the tutorial, I guess.

Reply:
see https://github.com/remindmodel/remind/pull/705

Reply:
Thanks for the explanation!